IntegrationTests/Services/AWSSTSIntegrationTests/STSWebIdentityAWSCredentialIdentityResolverTests.swift

```diff
@@ -82,24 +102,26 @@ class STSWebIdentityAWSCredentialIdentityResolverTests: XCTestCase {
 
     override func tearDown() async throws {
         // Delete inline identity policy of the role
-        try await deleteInlineRolePolicy()
+        try? await deleteInlineRolePolicy()
 
         // Delete role
-        _ = try await iamClient.deleteRole(input: DeleteRoleInput(roleName: roleName))
+        _ = try? await iamClient.deleteRole(input: DeleteRoleInput(roleName: roleName))
 
         // Delete Cognito identity pool
-        _ = try await cognitoIdentityClient.deleteIdentityPool(
+        _ = try? await cognitoIdentityClient.deleteIdentityPool(
             input: DeleteIdentityPoolInput(identityPoolId: identityPoolId)
         )
 
         // Delete token file
-        try deleteTokenFile()
+        try? deleteTokenFile()
+
+        try await super.tearDown()
```

Comment:
`super` is called during tear down.

Also, `try?` is used on throwing teardown operations so that the remainder of teardown may proceed if one part fails.

Reply:
Question: What's the point of `super.tearDown()` here? Isn't it just a no-op in `XCTestCase`?

Reply:
In general I call `super` in overrides unless I'm explicitly choosing to not call `super`; i.e. replacing & not just augmenting the current behavior.

(FWIW not calling `super` isn't wrong, but it's saved me trouble on refactors in the past)